datafusion/sql/src/planner.rs

```diff
@@ -460,8 +460,13 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 op,
                 left,
                 right,
-                all,
+                set_quantifier,
             } => {
+                let all = match set_quantifier {
```

Comment:
sqlparser now distinguishes between `ALL` , `DISTINCT` and `not specified` (previously, not specified left `all=false`)